var/spack/repos/builtin/packages/flux-pmix/package.py

```diff
@@ -18,6 +18,7 @@ class FluxPmix(AutotoolsPackage):
     maintainers("grondo")
 
     version("main", branch="main")
+    version("0.4.0", sha256="f7f58891fc9d9a97a0399b3ab186f2cae30a75806ba0b4d4c1307f07b3f6d1bc")
```

Comment:
This version of flux-pmix requires flux-core >= 0.49.0. Is there some way to specify that here?

Reply:
@grondo I thought we already had:

```
    depends_on("flux-core@0.49.0:", when="@0.3.0:")
```
to say it's required for 3 and up?

Reply:
oh we need to pin it to that, I see. Not go further up!

Reply:
Actually can we do:

```
    depends_on("flux-core@0.49.0", when="@0.3.0:0.4.0")
```
?

Reply:
Yes, you can do a range but why not drop the `.0` from the end of each version in the `when` argument?

Reply:
Looks like we need to add more examples in the docs for things like this.